jenkins/agent-ctl.sh

```diff
@@ -381,9 +381,9 @@ INSTANTIATE_DONE
 
 reboot-agent() {
   echo "Rebooting ${INSTANCE}..."
-  gcloud compute ssh "${INSTANCE}" sudo reboot || gcloud compute instances reset "${INSTANCE}"
+  gcloud compute ssh "${INSTANCE}" --command="sudo reboot || gcloud compute instances reset '${INSTANCE}'"
```

Comment:
this should be `--command='sudo reboot' || gcloud ...` - the right-hand branch runs locally, not remotely.